src/main/java/com/cs56fitnessapp/models/workout/FitnessFormulas.java

```diff
@@ -124,15 +128,15 @@ public static int calorieOffset(int progressPace) {
 
     /**
      * Using MET (Metabolic Equivalent of Task) constant and the formula
-     * Kcal ~= METS * bodyMassKg * timePerformingHours
+     * Kcal ~= met_Value * bodyMassKg * timePerformingHours
      * to calculate caloric value of activity
-     * @param MET_value Metabolic Equivalent of Task
+     * @param met_Value Metabolic Equivalent of Task
      * @param bodyMassKg user's weight
      * @param timePerformingHours time of the workout in hrs
      * @return caloric value of activity using MET constant
      */
```

Comment:
Thank you for correcting `MET_value`  to more proper formatting. Correct formatting according to the Google Style guide is the following: `metValue` - let's stick with it.